File: 01. JAVA/08. Operadores lógicos/Operadores logicos.java
package prueba;

public class Prueba {

    public static void main(String[] args) {
       
        /* Y = && = Las dos proposiciones deben ser verdaderas*/
        System.out.println((true && true)); //true
        System.out.println((true && false)); //false
        System.out.println((false && true)); //false
        System.out.println((false && false)); //false

        /* O = || = Una de las proposiciones debe ser verdadera*/
        System.out.println((true || true)); //true
        System.out.println((true || false)); //true
        System.out.println((false || true)); //true
        System.out.println((false || false)); //false

        /* NO = ! = Niega el valor de verdad */
        System.out.println(!(true)); //false
        System.out.println(!(false)); //true
        
    }
    
}

File: 05. Lenguajes no clasificados/01. C#/08. Operadores lógicos/Operadores logicos.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace prueba
{
    class Program
    {
        static void Main(string[] args)
        {

            /* Y = && = Las dos proposiciones deben ser verdaderas*/
            Console.WriteLine((true && true)); //true
            Console.WriteLine((true && false)); //false
            Console.WriteLine((false && true)); //false
            Console.WriteLine((false && false)); //false

            /* O = || = Una de las proposiciones debe ser verdadera*/
            Console.WriteLine((true || true)); //true
            Console.WriteLine((true || false)); //true
            Console.WriteLine((false || true)); //true
            Console.WriteLine((false || false)); //false

            /* NO = ! = Niega el valor de verdad */
            Console.WriteLine(!(true)); //false
            Console.WriteLine(!(false)); //true

            Console.In.ReadLine();

        }
    }
}
